src/middlewares/error.middleware.ts

```diff
@@ -33,12 +33,16 @@ const csrfErrorHandler = (err: CsrfError | Error, req: Request, res: Response, n
  * Use this error handler by calling next(e) from within a controller
  * Always keep this as the last handler in the chain for it to work.
  */
-const errorHandler = (err: Error, req: Request, res: Response, _next: NextFunction) => {
+const error = (err: Error, req: Request, res: Response, _next: NextFunction) => {
```

Comment:
This does look more like an error handler than simply an error and its current name aligns with the `csrfErrorHandler` definition in this class. Could it be called `generalErrorHandler` maybe?

Reply:
I can change it by generalErrorHandler or something else
But pageNotFound is not call handler as well

Reply:
generalErrorHandler
or 
globalErrorHandler

need help from the british!!!

Reply:
i vote for globalErrorHandler

Reply:
updated